posthog/otel_instrumentation.py

```diff
@@ -135,6 +137,39 @@ def initialize_otel():
                 "otel_instrumentation_attempt", instrumentor="AIOKafkaInstrumentor", status="error", exc_info=e
             )
 
+        def client_span_callback(span, response):
+            if not span or not span.is_recording():
+                return
+
+            key = "url.full" if span.attributes.get("url.full") else "http.url"
+            if key not in span.attributes:
+                return
+
+            url = span.attributes.get(key)
+            if not url:
+                return
+
+            url_parts = url.split("?")
```

Comment:
i think this lets the traces collapse down to domain and path
instead of seeing different trace names based on the query parameter